expandybird/test/templates/helpers/common.py

```diff
@@ -1,5 +1,3 @@
-# Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
These are just unit test files, not actual code, so I figured adding the standard comment was a bit overkill.